packages/celotool/src/e2e-tests/governance_tests.ts

```diff
@@ -323,6 +340,55 @@ describe('governance tests', () => {
     return tx.send({ from: account, ...txOptions, gas })
   }
 
+  describe('when adding any block', () => {
```

Comment:
Nit: Can you move this test below `Validators.numberValidatorsInCurrentSet()` to minimize the diff size?